examples/testprograms/http/http.star

```diff
@@ -4,10 +4,19 @@ load("env", "MEOW")
 print(MEOW)
 
 def on_http_get():
-    resp, err = catch(lambda: http1.get("https://httpbin3212.org/status/404"))
+    resp, err = http1.get("https://httpbin3212.org/status/404", ak.callopts(catch=True))
+
+    # also works:
+    # resp, err = catch(lambda: http1.get("https://httpbin3212.org/status/404"))
+    # resp, err = catch(http1.get, "https://httpbin3212.org/status/404")
+    # resp, err = http1.get("https://httpbin3212.org/status/404", ak = ak.callopts(catch=True))
+    # resp, err = http1.get("https://httpbin3212.org/status/404", ak_catch=True)
+    # resp, err = http1.get("https://httpbin3212.org/status/404", ak = {"catch": True})
+
     print(resp, err)
     print(err.op)
 
+
```

Comment:
Nit: Not sure if this line was added by mistake. Unlike Google Python style, Starlark style is to have only 1 empty line between top-level blocks (at least according to Google's Bazel formatter).